src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java

```diff
@@ -77,12 +77,16 @@ public ClaimantResponseContentProvider(
                 content.put("defendantAdmissionAccepted", "I accept this amount");
                 ResponseAcceptation responseAcceptation = (ResponseAcceptation) claimantResponse;
                 content.putAll(responseAcceptationContentProvider.createContent(responseAcceptation));
-                content.put("formaliseOption", responseAcceptation.getFormaliseOption().getDescription());
 
-                claim.getTotalAmountTillDateOfIssue().ifPresent(totalAmount -> content.put("totalAmount",
-                    formatMoney(totalAmount.subtract(claimantResponse.getAmountPaid().orElse(BigDecimal.ZERO)))));
+                claim.getTotalAmountTillDateOfIssue().ifPresent(
```

Comment:
We normally prefer functional statements to be as simple as possible, so you could do the processing of the `totalAmount` in a prior link in the chain:
```
claim.getTotalAmountTillDateOfIssue()
    .map(totalAmount -> totalAmount.subtract(claimantResponse.getAmountPaid().orElse(BigDecimal.ZERO)))
    .map(Formatting::formatMoney)
    .ifPresent(formattedAmount -> content.put("totalAmount", formattedAmount));
```